src/features/game/dashboard/GameDashboardTypes.ts

```diff
@@ -0,0 +1,8 @@
+export enum DashboardPage {
+  Log = 'Log',
+  Tasks = 'Tasks'
```

Comment:
`Collectibles` and `Achievements` should not be completely removed since they are both working features that can be used (even though they weren't used very much last semester). Also we might want to phase out AwardMenu in this case.

Note that AwardMenu is also used in RoomPreview, so it'll need to toggle Log and Tasks on or off.

Reply:
Added the collectibles and achievements into the dashboard.
Also added the dashboard and toolbar into the room preview scene. The dashboard now displays different pages depending on what pages have been specified in the dashboard constructor.